index.rst

```diff
@@ -2,24 +2,180 @@
 
 .. sectnum::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
 
-.. TODO: Delete the note below before merging new content to the main branch.
+Abstract
+========
 
-.. note::
+Continuous Integration and Continuous Delivery (CI/CD) are at the core of Telescope and Site Software requirement verification strategy.
 
-   **This technote is a work-in-progress.**
+The idea rely on continuously testing new and existing software with a well defined and well understood set of unit and integration tests.
+It also requires frequently deploying new software as it is made available to users both in testing and production environments. 
+This ensures that, as new functionality is included in the system, we are continuously adding verification steps while guaranteeing that these new features do not introduce unexpected errors.
 
-Abstract
-========
+Although a large portion of requirements/features can be tested in a CI/CD environment, there are some cases where customized tests using the production environment are necessary.
+These includes, but are not limited to, tests that are related to hardware or requirements that are indirectly related to software requirements.
+
+Finally, we must also be able to provide requirements verification artifacts, that can be used to certify that they were successfully verified.
+Here, we detail the various strategies and tools developed to support software requirement verification for Telescope and Site Software.
+
+Introduction
+============
+
+Requirement verification is an important process in the delivery of any project, and it consists of certifying that a project delivers what was required of it in the first place.
+In other words, requirement verification provides answers to the question
+
+> Have we built what we were asked to build?
+
+Because software is, by definition, extremely malleable, requirements verification can become quite challenging.
+To begin with, Telescope and Site Software (as most software teams in industry at large) adopts an Agile methodology, which, in simple terms, means we are constantly working on the most useful features and continuously delivering new functionality to users.
```

Comment:
"in simple terms" sounds a bit demeaning :)
Consider:
Telescope and Site Software (as most software teams in industry at large) utilizes an Agile methodology, which means we are continually adapting to focus on the most useful features and delivering new functionality to users at a high cadence.